### PR TITLE
owner: revoke on ctx.Done.

### PR DESCRIPTION
**Pull request by coocood**
Without `revoke`, when bootstrap finished or server exit, there is an owner entry left on etcd, so the new DDL cannot become the owner instantly.

The test depends on etcd, so I just did a manual test to verify that initialize cluster and restart the tidb-server can create table instantly.

### CODE REVIEW

**Comment by coocood** [comment]
@zimulala @tiancaiamao PTAL

**Comment by zimulala** [comment]
Please add some tests.

**Comment by coocood** [comment]
@zimulala PTAL

File: owner/manager.go [hunk 1]
```diff
@@ -203,12 +198,21 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.
...
 }
 
+func (m *ownerManager) revokeSession(logPrefix string, leaseID clientv3.LeaseID) {
+	// Revoke the session lease.
+	// If revoke takes longer than the ttl, lease is expired anyway.
+	cancelCtx, cancel := goctx.WithTimeout(goctx.Background(),
+		time.Duration(ManagerSessionTTL)*time.Second)
+	_, err := m.etcdCli.Revoke(cancelCtx, leaseID)
+	cancel()
+	log.Infof("%s break campaign loop, revoke err %v", logPrefix, err)
```
Comment (shenli) [inline owner/manager.go]: If err is nil, we do not need print it.

Comment (coocood) [reply owner/manager.go]: The old code prints the log for breaking campaign loop, not only for the error.

**Comment by zimulala** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**